docs/validations.stories.mdx

```diff
@@ -0,0 +1,133 @@
+* footer - a container for rendering "Save", "Cancel" and other custom buttons; it can also be made sticky if wanted
+* validation summary - graphical indicator of errors and warnings
+
+For more information check [Form in Storybook](https://carbon.sage.com/?path=/docs/design-system-form)
+
+## Formik overview
+
+> Formik keeps track of your form's state, and then exposes it (plus a few reusable methods and event handlers) to your
+> form via props. [🔗](https://jaredpalmer.com/formik/docs/overview#the-gist)
+
+Formik provides a small set of React components and hooks, of which the key ones are:
+
+* [**`<Formik>`**](https://jaredpalmer.com/formik/docs/api/formik)  –  The central component of Formik, which wraps an
+  HTML form, and accepts props such as
+  [`initialValues`](https://jaredpalmer.com/formik/docs/api/formik#initialvalues-values),
+  [`onSubmit()`](https://jaredpalmer.com/formik/docs/api/formik#onsubmit-values-values-formikbag-formikbag-void),
+  [`validate()`](https://jaredpalmer.com/formik/docs/api/formik#validate-values-values-formikerrors-values-promise-any)
+  and [`validationSchema()`](https://jaredpalmer.com/formik/docs/api/formik#validationschema-schema-schema).  It
+  provides form state information (such as
+  [`values`](https://jaredpalmer.com/formik/docs/api/formik#values-field-string-any),
+  [`dirty`](https://jaredpalmer.com/formik/docs/api/formik#dirty-boolean),
+  [`errors`](https://jaredpalmer.com/formik/docs/api/formik#errors-field-string-string) and
+  [`isValid`](https://jaredpalmer.com/formik/docs/api/formik#isvalid-boolean)) and action methods (such as
+  [`validateForm()`](https://jaredpalmer.com/formik/docs/api/formik#validateform-values-any-promise-formikerrors-values),
+  [`validateField()`](https://jaredpalmer.com/formik/docs/api/formik#validatefield-field-string-void) and
+  [`setValues()`](https://jaredpalmer.com/formik/docs/api/formik#setvalues-fields-field-string-any-void)).
+* [**`<Form>`**](https://jaredpalmer.com/formik/docs/api/form),
+  [**`<Field>`**](https://jaredpalmer.com/formik/docs/api/field),
+  [**`<ErrorMessage>`**](https://jaredpalmer.com/formik/docs/api/errormessage)  –  Helper components (powered by
+  [React Context](https://reactjs.org/docs/context.html)) which reduce boilerplate code.
+* [**`useField()`**](https://jaredpalmer.com/formik/docs/api/useField)  –  A hook to help with building reusable input
+  primitive components.
+
+**Note:** Formik is only intended to be used with
+[controlled components](https://reactjs.org/docs/forms.html#controlled-components), where the component's state is
+controlled by supplying a `value` prop (or a `checked` prop for `radio` and `checkbox`).  Technically it's possible to
+use Formik with [uncontrolled components](https://reactjs.org/docs/uncontrolled-components.html) (by not supplying a
+controlling `value`/`checked` prop, and only supplying `name` `onChange` `onBlur` props), but this is strongly
+discouraged :warning:
```

Comment:
This renders as the word warning, not the warning icon.
![image](https://user-images.githubusercontent.com/35449967/84669823-38cb9580-af1d-11ea-8e18-e5ea507e37e5.png)

Reply:
Viewed via Storybook in Chrome